docs/cicd_tools/common.md

```diff
@@ -0,0 +1,69 @@
+# Common module
+
+This module exposes helper functions that may be used (and shared) across all modules, or that don't
```

Comment:
I'd slim this down to "Helper functions that are shared and used by other modules."

Reply:
I like it, thanks!